lib/v8debugapi.js

```diff
@@ -189,7 +190,7 @@ module.exports.create = function(logger_, config_, fileStats_) {
       var num = breakpoints[breakpoint.id].v8Breakpoint.number();
       var listener = onBreakpointHit.bind(
           null, breakpoint, function(err) {
-            delete listeners[num];
+            listeners[num].enabled = false;
```

Comment:
Since this is an aync function might it be a good idea to check for the `[num]` property before accessing it in case the array is mutated by a bad commit sometime in the future? Example:
```
listeners[num].enabled = false
```
Check first:
```
if (typeof listeners[num] !== undefined) {
  listeners[num].enabled = false;
} 
```